tests/Query/FallbackExecutorTest.php

```diff
@@ -0,0 +1,224 @@
+<?php
+
+namespace React\Tests\Dns\Query;
+
+use React\Dns\Model\Message;
+use React\Dns\Query\FallbackExecutor;
+use React\Dns\Query\Query;
+use React\Promise\Promise;
+use React\Tests\Dns\TestCase;
+
+class FallbackExecutorTest extends TestCase
```

Comment:
```suggestion
final class FallbackExecutorTest extends TestCase
```

Reply:
Did take a look and decided against this because this only affects test classes that aren't part of our public API anyway. Also, none of the test classes are currently final, so it's more consistent as-is.